### DISCUSSION TITLE
How to show a webpage built using React on camera?

### DISCUSSION THREAD

**Discussion by ThenoobMario** [AxisCommunications/acap-native-sdk-examples#184] [Q&A] [answered]
Hello, 

I followed the process of deploying a webpage mentioned in issue #163. It states to create a simple HTML form with Javascript embedded into the HTML file. I have successfully done that and it works. 

Now to improve on the frontend of the application and add more features, I am creating a react-based frontend. However, once I build that react application, and store the build files inside the html directory for EAP building, I am not able to get the webpage to run. 

Where am I going wrong? I have attached the error as well. 

![Screenshot from 2024-01-05 17-04-17](https://github.com/AxisCommunications/acap-native-sdk-examples/assets/60312074/37587433-c991-4129-bc76-33434c2a31d9)

**Comment by pataxis** [comment]
Hi @ThenoobMario ,
it's hard to say anything from the error messages.
If you create a simple example that demonstrates the problem you have in a user repository with public visibility it would make it easier to test and find any possible problem.

**Reply by ThenoobMario** [reply, marked as answer]
Hi @pataxis,
Thanks for the reply. After some digging through I found out it was an issue with relative path building when creating the react build. It is solved now.
